doc/classes/OS.xml

```diff
@@ -189,26 +189,24 @@
 				for argument in OS.get_cmdline_args():
 				    if argument.contains("="):
 				        var key_value = argument.split("=")
-				        arguments[key_value[0].lstrip("--")] = key_value[1]
+				        arguments[key_value[0].trim_prefix("--")] = key_value[1]
 				    else:
 				        # Options without an argument will be present in the dictionary,
 				        # with the value set to an empty string.
-				        arguments[argument.lstrip("--")] = ""
+				        arguments[argument.trim_prefix("--")] = ""
 				[/gdscript]
 				[csharp]
-				var arguments = new Godot.Collections.Dictionary();
+				var arguments = new Dictionary&lt;string, string&gt;();
```

Comment:
Is this part correct @raulsntos? What does the general docs use here, should we suggest using non-compatible data types if the code doesn't make use of them with the engine?

Reply:
We do recommend using the BCL collections whenever possible, unless you need this collection to interact with the engine[^1]. I feel like this change is consistent with our recommendations.

There are probably many cases in the documentation where we use Godot collections even though we technically don't need to, some of them because they were ported directly from GDScript perhaps by contributors not very familiar with C# and I think having some example code is better than none.

[^1]: https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/c_sharp_collections.html#choose-a-collection